contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -139,8 +143,10 @@ impl Contract {
                 Some(amount) => amount,
                 None => panic!("Failed divide deposit to refund GameDeposit (GameResult::Tie)"),
             };
+            print!("refund ammount {}", refund_amount);
+            print!("reward.balance.0 {}", reward.balance.0);
             assert!(
-                refund_amount.checked_mul(PLAYERS_NUM as u128) < Some(reward.balance.0),
+                refund_amount.checked_mul(PLAYERS_NUM as u128) <= Some(reward.balance.0), //TODO: I think it must be smaller or equal since we allow zero fees
```

Comment:
```suggestion
                winner_reward <= reward.balance.0,
```

Reply:
And, I think we can move the check just above the `let refund_amount` declaration.